homeassistant/components/ps4/media_player.py

```diff
@@ -154,20 +154,20 @@ def check_region(self):
         if self._ps4.ddp_protocol is not None:
             # Request Status with asyncio transport.
             self._ps4.get_status()
-            if not self._ps4.connected and not self._ps4.is_standby:
+            if not self._ps4.connected and not self._ps4.is_standby\
+                    and self._state is not None:
```

Comment:
Why do we not connect if state is `None`?

Reply:
This was to make sure the PS4 is ready for connection. Reverted. Fixed library issue so this is not needed.